source/jsonnet/england-wales/household/blocks/visitor/visitor_usual_address_country.jsonnet

```diff
@@ -21,7 +21,7 @@ local rules = import 'rules.libsonnet';
         label: 'Current name of country',
         description: 'Enter your own answer or select from suggestions',
         mandatory: false,
-        suggestions_url: 'https://cdn.eq.census-gcp.onsdigital.uk/data/v1.0.0/countries-of-birth.json',
+        suggestions_url: 'https://cdn.eq.census-gcp.onsdigital.uk/data/{version}/gb/{language_code}/countries-of-birth.json',
```

Comment:
I think if we've added version to what we're replacing, we should consider whether the script should define the api url in one place. We could do something like:

 ```
        suggestions_url: '{suggestions_api_url}/countries-of-birth.json',
```

What do you think @MebinAbraham?

Reply:
Ya, I think that is a good idea to define the API URL in 1 place. The suggested pattern makes sense.

Reply:
Done, thanks!👍